src/main/java/util/Storage.java

```diff
@@ -0,0 +1,60 @@
+package util;
+
+import java.io.*;
+
+/**
+ * Storage class to help with loading and storing of Tasks in duke.txt.
+ * @author Merrick
+ */
+public class Storage {
+    private final String filePath;
+    private final String fileDir;
+    /**
+     * Constructor of Storage class
+     * @param fileDir Directory of duke.txt file.
+     * @param filePath Filepath of duke.txt file.
+     */
+    public Storage(String fileDir, String filePath) {
+        // check if fileDir and filePath are not empty
+        this.fileDir = fileDir;
+        this.filePath = filePath;
+    }
+
+    /**
+     * Loads the tasks stored in the duke.txt file.
+     * @return BufferedReader object that contains the list of tasks.
+     * @throws DukeException If IOException from reading duke.txt is encountered.
+     */
+    public BufferedReader load() throws DukeException {
```

Comment:
Might be better for the method to have a more descriptive name, such as `loadData()` or  `loadTasksFromFile`.